### ISSUE TITLE
RTL - Color picker HSV/RGB fields incorrectly mirrored

### ISSUE THREAD

**Issue by jmanke** [Esri/calcite-components#4238]
### Summary

The RGB and HSV fields are incorrectly RTL mirrored.

### Actual Behavior

![image](https://user-images.githubusercontent.com/19629809/158449379-67431b2e-2aee-4184-ac7a-0563c5669db7.png)

### Expected Behavior

The RGB and HSV fields should not be RTL mirrored, they should be in R,G,B and H,S,V order like on EN language.

### Reproduction Sample

https://codepen.io/jmanke/pen/NWXqBRb

### Reproduction Steps

1. Set dir to 'rtl' in the root element
2. Observe RGB and HSV fields are incorrectly RTL mirrored.

### Reproduction Version

@esri/calcite-components@latest

**Comment by jcfranco** [comment]
@anveshmekala I _think_ the layout within inputs is correct in RTL, it's the order of the inputs that needs to be fixed. cc @macandcheese

**Comment by jmanke** [comment]
@anveshmekala @jcfranco Yes @jcfranco is correct

**Comment by benelan** [comment]
verified on `beta.81`
